src/UWP/ArcGISRuntime.UWP.Viewer/MainPage.xaml

```diff
@@ -79,8 +78,8 @@
                     <VisualState.Setters>
                         <Setter Target="ApplicationTitle.FontSize" Value="16"/>
                         <Setter Target="ApplicationTitle.Margin" Value="12,0,0,0"/>
-                        <Setter Target="SearchToggleButton.Visibility" Value="Collapsed"/>
-                        <!--<Setter Target="SearchBox.Visibility" Value="Collapsed"/>-->
+                        <Setter Target="SearchToggleButton.Visibility" Value="Visible"/>
+                        <Setter Target="SearchBox.Visibility" Value="Collapsed"/>
```

Comment:
The searchbox collapses after each keystroke when we're in the narrowest visual state. Bug?